### ISSUE TITLE
"I wait to see a navigation bar titled" bug

### ISSUE THREAD

**Issue by svdo** [calabash/calabash-ios#212]
The predefined step "I wait to see a navigation bar titled [...]" doesn't have the expected result when there is a back button.

Scenario:
- I have a navigation controller; the root view has the title "Library"
- I navigate to a sub controller, so that there is a back button labeled "Library" and a navigation title which is NOT "Library"
- I test for "Then I wait to see a navigation bar titled "Library"

Expected result: returns _false_ because I'm in the sub controller which has a different title than "Library"
Actual result: returns _true_ because the implementation of this step finds the label "Library" on the back button.

In other words: the step should ONLY check the title, not any UINavigationItemButtonView that may also be present.
